static/src/javascripts/projects/commercial/modules/article-body-adverts.js

```diff
@@ -19,6 +19,8 @@ type AdSize = {
 
 type Sizes = { desktop: Array<AdSize> };
 
+const isPaidContent = config.get('page.isPaidContent');
```

Comment:
Minor: add fallback value for `config.get` to avoid potential undefined?

`const isPaidContent: boolean = config.get('page.isPaidContent', false);`

Reply:
Yes!